README.md

```diff
@@ -20,6 +20,10 @@ For convinience you can use the shell you like the most, you can choose the foll
 - bash
 - sh
 
+### Build locally
+
+```sh
+docker build -t debug:local . -f src/Dockerfile
```

Comment:
I can't propse to close the code block 🙄

Reply:
done 👍